packages/e2e/features/ui/components/storage/storage-browser/action-menu.feature

```diff
@@ -139,3 +139,71 @@ Feature: Create folder with Storage Browser
     # verify all files are deleted
     Then I see "No files"
 
+  @react
+  Scenario: Upload file shows a Network error if offline
+      When I type my "email" with status "CONFIRMED"
+      Then I type my password
+      Then I click the "Sign in" button
```

Comment:
Nit: this is entirely non-blocking of this PR, but would it fit for there to be step specifically for signing into the storage browser which would just be equivalent to these three steps? Having every test start with 3 - 4 steps to get to the right starting place can really cause these files to get large and less straightforward to parse